### ISSUE TITLE
`cmp` setup comes with a warning on the line that sets: `cmp.mapping.complete()`

### ISSUE THREAD

**Issue by fernandocanizo** [nvim-lua/kickstart.nvim#95]
**Describe the bug**
`cmp` setup comes with a warning on the line that sets: `cmp.mapping.complete()`

**To Reproduce**
1. Kickstart a clean Neovim configuration as explained in the README
2. Use `]d` to go to the warning or open the quickfix window with `<leader>q`

**Desktop (please complete the following information):**
- OS: Archlinux
- Terminal: urxvt

** Neovim Version **
 - v0.8.1

Warning says
```
init.lua|397 col 21-43 warning| This function requires 1 argument(s) but instead it is receiving 0.
```

Help for the function shows an example to avoid completion in a specific circumstance, but doesn't explain the parameters. There's no help for `cmp.ConfigSchema` nor `cmp.ContextReason`.

Putting an empty table fixes the warning, but I don't know if that breaks some functionality:
```
['<C-Space>'] = cmp.mapping.complete({}),
```

**Comment by mischavandenburg** [comment]
I get the same warning.

**Comment by mlabrkic** [comment]
After this I have no more warning:
    ['<C-Space>'] = cmp.mapping.complete({}),  --  {}

warning| This function requires 1 argument(s) but instead it is receiving 0.

  mapping = cmp.mapping.preset.insert {
    ['<C-d>'] = cmp.mapping.scroll_docs(-4),
    ['<C-f>'] = cmp.mapping.scroll_docs(4),
    ['<C-Space>'] = cmp.mapping.complete({}),  -- because warning (1 argument):  {}
    ['<CR>'] = cmp.mapping.confirm {
      behavior = cmp.ConfirmBehavior.Replace,
      select = true,
    },

**Comment by tjdevries** [comment]
Fixed by #178
